modules/gui/src/pointcloud/vpColorBlindFriendlyPalette.cpp

```diff
@@ -0,0 +1,173 @@
+#include <visp3/gui/vpColorBlindFriendlyPalette.h>
+#include <visp3/core/vpIoTools.h>
+
+std::vector<std::string> vpColorBlindFriendlyPalette::g_paletteNames =
+{
+  "black"     ,
+  "orange"    ,
+  "sky-blue"   ,
+  "green"     ,
+  "yellow"    ,
+  "blue"      ,
+  "vermillon" ,
+  "purple"    ,
+  "unknown"
+};
+
+std::vector<vpColor> vpColorBlindFriendlyPalette::g_palette = {
+  vpColor(0,0,0),       // Black = 0,
+  vpColor(230,159,0),   // Orange = 1,
+  vpColor(86,180,233),  // SkyBlue = 2,
+  vpColor(0,158,115),   // Green = 3,
+  vpColor(240,228,66),  // Yellow = 4,
+  vpColor(0,114,178),   // Blue = 5,
+  vpColor(213,94,0),    // Vermillon = 6,
+  vpColor(204,121,167), // Purple = 7,
+  vpColor(255,255,255)  // COUNT = 8
```

Comment:
Why not using colors already defined in ViSP `vpColor::black, vpColor::orange...`

Reply:
It is because the colors that are color-blind friendly have particular RGB values to ensure that they are distinguishable